pkg/kubectl/cmd/get.go

```diff
@@ -94,6 +95,23 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		return err
 	}
 
+	if len(args) == 0 {
+		fmt.Fprint(out, `
+You must specify the type of resource to get. Valid resource types include:
+   * pods (aka 'po')
+   * replicationcontrollers (aka 'rc')
+   * services
```

Comment:
this is marked as 'deprecated will remove before 1.0' in https://github.com/GoogleCloudPlatform/kubernetes/blob/5520386b180d3ddc4fa7b7dfe6f52642cc0c25f3/pkg/kubectl/kubectl.go#L115 so I didn't include it